From 798dc162e494d73692e5f2da34730c46679e9f51 Mon Sep 17 00:00:00 2001
From: Guoqing Jiang <b22156@freescale.com>
Date: Thu, 5 Nov 2009 15:43:58 +0800
Subject: [PATCH 128/442] ENGR00118084 MX25: Change Makefile to build-in
 camera capture of v4l2 unit test.

1. Change Makefile to build-in camera capture of v4l2 unit test.
2. Remove warning message.

Signed-off-by: Guoqing Jiang <b22156@freescale.com>
---
 test/mxc_v4l2_test/Makefile           | 3 ++-
 test/mxc_v4l2_test/csi_v4l2_capture.c | 1 -
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/mxc_v4l2_test/Makefile b/test/mxc_v4l2_test/Makefile
index e228830..e66b97f 100644
--- a/test/mxc_v4l2_test/Makefile
+++ b/test/mxc_v4l2_test/Makefile
@@ -7,7 +7,8 @@ OBJS = 	$(OBJDIR)/autorun-v4l2.sh \
 	$(OBJDIR)/mxc_v4l2_still.out \
 	$(OBJDIR)/mxc_v4l2_tvin.out
 ifeq ($(PLATFORM), IMX25_3STACK)
-OBJS += $(OBJDIR)/csi_v4l2_overlay.out
+OBJS += $(OBJDIR)/csi_v4l2_overlay.out \
+	$(OBJDIR)/csi_v4l2_capture.out
 else
 OBJS += $(OBJDIR)/mxc_v4l2_overlay.out \
 	$(OBJDIR)/mxc_v4l2_capture.out
diff --git a/test/mxc_v4l2_test/csi_v4l2_capture.c b/test/mxc_v4l2_test/csi_v4l2_capture.c
index cd7ba25..4f76c1d 100644
--- a/test/mxc_v4l2_test/csi_v4l2_capture.c
+++ b/test/mxc_v4l2_test/csi_v4l2_capture.c
@@ -120,7 +120,6 @@ int v4l_capture_setup(void)
 {
         char v4l_device[100] = "/dev/video0";
         struct v4l2_format fmt;
-        struct v4l2_control ctrl;
         struct v4l2_streamparm parm;
         int fd_v4l = 0;
 
-- 
1.8.0

